test/unit/modules/observer/observer.spec.js

```diff
@@ -0,0 +1,320 @@
+import Vue from 'vue'
+import {
+  Observer,
+  observe,
+  set as setProp,
+  del as delProp
+} from '../../../../src/core/observer/index'
```

Comment:
can just use `core/observer/index` because the build has [these aliases](https://github.com/vuejs/vue/blob/next/build/alias.js)